week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,8 +5,12 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
-
+function tidyUpString(arr) {
+  function stringarr(strArr) {
```

Comment:
It's normally better to put functions outside of each other so this code would be like this:

```
function tidyUpString(arr) {
  return arr.map(stringarr);
}

function stringarr(strArr) {
  return strArr.replace(/\//g,'').trim().toLowerCase();
}
```

Reply:
This can get really confusing if you start putting functions inside each other!